tools/syz-declextract/syz-declextract.cpp

```diff
@@ -49,6 +52,18 @@ struct NetlinkOps {
   std::optional<std::string> policy;
 };
 
+bool endsWith(const std::string_view str, const std::string end) {
```

Comment:
Immutable strings are typically passed by const reference to avoid unnecessary memory allocations and data copies.
But would std::string_view would work for the second arg of these functions? I would assume it should be constructable from string literals. That would be even better type.

Reply:
Yeah, makes sense! I think I just switched up the parameters last minute and forgot to change the type. I'll fix it.

Reply:
Done!